pkg/controller/storagecluster/reconcile.go

```diff
@@ -1190,6 +1190,11 @@ func newCleanupJob(sc *ocsv1.StorageCluster) *batchv1.Job {
 		"app": "ceph-toolbox-job-${FAILED_OSD_ID}",
 	}
 
+	const purgeOSDScript = `set -x
+if [[ \"$osd_status\" == \"up\" ]]; then echo \"OSD ${FAILED_OSD_ID} is up and running. Please check if you entered correct ID of failed osd!\"; else echo \"OSD ${FAILED_OSD_ID} is down. Proceeding to mark out and purge\"
```

Comment:
Do you really need to escape the quotes? Since it's inside the multi-line string, that's not needed for the golang string. Was it failing in the job without the quotes?

Reply:
Would be helpful for readability if the `else` is also on the next line. In other words, the pattern:
```
if <condition>; then
  <action>
else
  <action>
fi
```

Reply:
ack